pkg/segment/structs/segstructs.go

```diff
@@ -812,26 +799,19 @@ func (ss *NumericStats) Merge(other *NumericStats) {
 	if other == nil {
 		return
 	}
-	switch ss.Min.Ntype {
+
+	switch ss.Sum.Ntype {
 	case utils.SS_DT_FLOAT:
 		if other.Dtype == utils.SS_DT_FLOAT {
-			ss.Min.FloatVal = math.Min(ss.Min.FloatVal, other.Min.FloatVal)
-			ss.Max.FloatVal = math.Max(ss.Max.FloatVal, other.Max.FloatVal)
 			ss.Sum.FloatVal = ss.Sum.FloatVal + other.Sum.FloatVal
```

Comment:
Why is it correct to remove these lines?

Reply:
We would be deprecating this in the next PR when we move to new SST format. Have kept it in segStats to keep this PR simpler, or else would need to update the SST writer and reader along with this.